components/NavigationHeader/index.tsx

```diff
@@ -51,6 +51,17 @@ export const NavigationHeader: React.VFC<{ path: string }> = ({ path }) => {
           <a>4. User Preferences</a>
         </li>
       </Link>
+      <Link href={`/benefit-status?id=${userData}`} passHref>
```

Comment:
Does this pass the href down correctly? I though the correct implementation required the `<a>` to be it's direct child

Reply:
yea, looks like it does